kubespawner/spawner.py

```diff
@@ -1559,20 +1559,20 @@ def poll(self):
             yield self.pod_reflector.first_load_future
         data = self.pod_reflector.pods.get(self.pod_name, None)
```

Comment:
The docstring on the `PodReflector.pods` attribute should probably be updated because it talks about the value being an object representation which it no longer is.

https://github.com/jupyterhub/kubespawner/pull/424/files#diff-3082573191d96b79e2ec71986d4ef3e2R69

Reply:
Good catch. I'll get all of these comments and docstrings fixed.